web/.dependency-cruiser.js

```diff
@@ -342,6 +342,12 @@ module.exports = {
       from: { path: "src/scripts" },
       to: { path: "../shared" },
     },
+    {
+      from: { path: "../content/build.ts" },
+      to: { path: "../shared" },
+      comment:
+        "Allow content build script to import from shared (build script is standalone executable)",
+    },
```

Comment:
I didn't know you could leave comments!

As it says, `dependency-cruiser` thinks us importing from `shared/` which then imports from `content/` (the .json files are imported directly) is bad and an indicator of a circular dependency.

It's not wrong. This is a circular dependency.

But it's not an issue here since the content build script is standalone, and nothing in `shared/` references the content build script thankfully.